de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/SarosSessionManager.java

```diff
@@ -114,6 +115,16 @@
 
     private final ProjectNegotiationObservable currentProjectNegotiations;
 
+    /** to queue resources for the next project negotiation **/
```

Comment:
That is what the HashMap is supposed to be used for. The description is not really helpful.